### DISCUSSION TITLE
Set Default Easing Preset

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#452] [Ideas]
I've been using ToonSquid to make Storytime Animations for a few weeks now, I'm on my second one right now, and I've noticed that whenever I create a new keyframe, it automatically sets it to linear, meaning I have to go in and change it to hold manually. It'd be nice if there were an option to where I could have the easing automatically set to whatever I need for my workflow, in this case, make it to where the keyframes would automatically be set to the hold preset. And considering that my current video is gonna be about 6 minutes long, something like this would speed up my workflow for these Storytime animations.

**Comment by azulika** [comment]
seconded. it would be very nice if this is implemented
